orte/mca/odls/base/odls_base_default_fns.c

```diff
@@ -258,7 +258,7 @@ int orte_odls_base_default_get_add_procs_data(opal_buffer_t *buffer,
         }
         /* if we didn't rollup the connection info, then we have
          * to provide a complete map of connection info */
-        if (!orte_static_ports && !orte_fwd_mpirun_port) {
+        if (!orte_fwd_mpirun_port) {
```

Comment:
I think we have to eliminate this code section as well since we always have to provide the map as the daemons now always "phone home"

Reply:
will be in the next commit.